dist/privacy-sandbox.js

```diff
@@ -0,0 +1,91 @@
+[privacy-sandbox]
+// Topics API Usage Reference: https://developers.google.com/privacy-sandbox/relevance/topics/demo#the-topics-api-demo
+// Header Usage Reference: https://developers.google.com/privacy-sandbox/relevance/topics/demo#the-topics-api-demo
+
+let requests = $WPT_BODIES;
+const testURL = new URL($WPT_URL);
+let firstPartyDomain = testURL.hostname;
+
+result = {
+    topicsAvailable: document.featurePolicy.allowsFeature('browsing-topics'),
```

Comment:
Any idea when this would be false?

Reply:
I think when FeaturePolicy for "browsing-topics" is either not set or used. I used this based on your suggestion.

Reply:
@pmeenan @max-ostapenko To enable these Privacy Sandbox features we can use command-line flags. For Chrome or Chromium-based browsers, these are the necessary flags:

**Topics API** ([cf](https://developers.google.com/privacy-sandbox/relevance/topics/developer-guide#view-api-information))
`--enable-features=BrowsingTopics`
`--enable-blink-features=BrowsingTopics`

**Protected Audience (FLEDGE)** ([cf](https://developers.google.com/privacy-sandbox/relevance/protected-audience-api#try-fledge))
`--enable-features=InterestGroupStorage`
`--enable-blink-features=InterestGroupStorage`

An example of launching the browser with the above flags could be the following command:
`chrome --enable-features=BrowsingTopics,InterestGroupStorage --enable-blink-features=BrowsingTopics,InterestGroupStorage`

Reply:
You need the following command line arguments:

```
--enable-features=BrowsingTopics,InterestGroupStorage,PrivacySandboxAdsAPIsOverride
```

However, I'm not sure what you hope to get out of this?

You won't be able to see how many sites enable this, since you've force enabled this, so it will be ALL sites.
You won't be able to see what topics or interest groups are set, since that is only set on second visit and Chrome basically freshly install for each instance of the run, so these will be empty each time.

Am I missing something?

Reply:
@tunetheweb We are seeing if TopicsAPI is used by embeded advertisers and trackers on a page via Javascript or Headers. The usage in Javascript is detected using string search but for Headers, the TopicsAPI will have to be enabled. @max-ostapenko Feel free to add anything missing.

Reply:
No API - no headers.

1. So true, with flags added we may remove the `topicsAvailable` as it will be always `true`.
2. We apply the flags to activate API. It will signal 3rd party to use it, and we'll be able to track it via headers.
3. We don't expect topics to be meaningful in the lab tests (even in my own browser I usually see empty list `();p=P0000000000000000000000000000000`), but at least headers to be present.

I've seen only *.doubleclick.net receiving and observing topics. We'll be able to see the bigger picture once we have the crawl data.

Reply:
Features have been enabled on the agent now.